client/src/Components/Pages/Dashboard/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import Girl from './PersonalInfo'
+import Find from './ActiveJobs'
+import Prevjobs from './PreviuosJobs'
+import News from './News'
+import Active from './MyActivety'
+
+import './style.css';
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div className="grand-div">
+        <header className="App-header">
```

Comment:
This is probably something you'll have to talk about with the whole team, but it might be a better component organization to have the project's header and footer exist in App.js, so that there's just one component that's responsible for rendering them, instead of having to add them to the dashboard, the profile page, the landing page, and so on.

If you are going to use react-router for navigation, then App.js would have the header, the router switch, and the footer.  That's how I usually organize my apps.

Reply:
Sorry! I just noticed that the landing page in the figma prototype does not have the same header / footer as the other pages, so that means my advice was wrong, putting the header and footer into App.js won't help.

There's an advanced React concept called[ Higher-Order Components](https://reactjs.org/docs/higher-order-components.html) that help with situations like this, where you don't want to have to add a header and footer to every single page's render function.  It lets you export components wrapped with other things.  Maybe an improvement for another day :)

Reply:
Entirely my fault as I changed it but didn’t change all the others. 
Thank you for all your help and contributions. 
Marie-Clare (product owner)

Reply:
Done